flytekit/clis/sdk_in_container/serialize.py

```diff
@@ -31,9 +31,15 @@
 _DOMAIN_PLACEHOLDER = "{{ registration.domain }}"
 _VERSION_PLACEHOLDER = "{{ registration.version }}"
 
+# During out of container serialize the absolute path of the entrypoint at serialization time won't match the
+# in-container value at execution time. The following default value is used to provide the in-container entrypoint path
+# but can be optionally overridden at serialization time based on the installation of your flytekit virtualenv.
+_DEFAULT_IN_CONTAINER_ENTRYPOINT = "/opt/venv/bin/entrypoint.py"
```

Comment:
we probably want to say, default_flytekit_entrypoint_fileloc

Reply:
done